string object
---
float32[] size
bool radius